docker/bash/commands/docker.sh

```diff
@@ -1,20 +1,25 @@
 #!/usr/bin/env bash
 
-_get_node_containers() {
+_get_containers_for_installing_node_modules() {
   # log-hub and hybridge containers are still using the old legacy nodejs docker image
   local ancestor_containers=$(docker ps --format '{{.Names}}' -f ancestor=pc-nodejs)
-  local pattern_containers=$(docker ps --format '{{.Names}}' | grep -E '^pc-core|^pc-fsa|^pc-fia|^pc-moncomptepro|^pc-dpa|^pc-exploitation-fca-low')
+  local pattern_containers=$(docker ps --format '{{.Names}}' | grep -E '^pc-exploitation-fca-low')
 
   echo -e "${ancestor_containers}\n${pattern_containers}" | sort | uniq
 }
 
 # Find which nodejs containers are running and store it into $NODEJS_CONTAINERS
-_get_running_containers() {
-  local raw_nodejs_containers=$(_get_node_containers)
-  local raw_all_containers=$(docker ps --format '{{.Names}}')
+_get_node_containers_to_start() {
+  local ancestor_containers=$(docker ps --format '{{.Names}}' -f ancestor=pc-nodejs)
```

Comment:
As not everyone in the team is fluent in bash (I'm not hihi), could you write extra comments on these lines to help understand how this function works? And/or rename the variables to be super explicit about what they're supposed to contain?